apps/test/unit/gamelab/animationListModuleTest.js

```diff
@@ -563,4 +564,34 @@ describe('animationListModule', function () {
       expect(store.getState().animationList.pendingFrames.props).to.deep.equal(libraryAnimProps);
     });
   });
+
+  describe('action: save animation', function () {
+    beforeEach(function () {
+      this.xhr = sinon.useFakeXMLHttpRequest();
+      let requests = this.requests = [];
+      this.xhr.onCreate = function (xhr) {
```

Comment:
I personally prefer not using `this` in tests - I'm a little fuzzy what it's bound to.  Instead, declare locals in the `describe` block.

```javascript
describe('action: save animation', function () {
  let xhr, requests;
  beforeEach(function () {
    xhr = sinon.useFakeXMLHttpRequest();
    requests = [];
    xhr.onCreate = xhr => requests.push(xhr);
  });

  // Test cases...
});
```